runtime/src/snapshot_utils.rs

```diff
@@ -979,7 +979,6 @@ pub fn process_accounts_package_pre(
         let (hash, lamports) = AccountsDb::calculate_accounts_hash_without_index(
             &accounts_package.storages,
             thread_pool,
-            accounts_package.cluster_type,
```

Comment:
Hey, @brooksprumo , in another pr you could consider removing cluster_type from accounts_package maybe. Maybe we don't need it for anything else?

Reply:
Will do!

Reply:
@ryoqun Do you know if `cluster_type` is used for anything still?